packages/@react-aria/aria-modal-polyfill/test/index.test.js

```diff
@@ -25,7 +25,7 @@ import {watchModals} from '../';
 describe('watchModals', () => {
   let matchMedia;
   beforeAll(() => {
-    jest.useFakeTimers();
+    jest.useFakeTimers('legacy');
   });
   afterAll(() => {
     jest.useRealTimers();
```

Comment:
Does this `useRealTimers` need to be removed? Noticed that it was removed in the other tests.

Reply:
doesn't need to be removed per-se, it just doesn't do anything in an afterAll at the top, but it won't hurt anything